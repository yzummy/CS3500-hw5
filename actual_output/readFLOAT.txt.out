"Enter a float (e.g., -2.71): "

---- Completed parsing ----

 Value of the expression is: 
