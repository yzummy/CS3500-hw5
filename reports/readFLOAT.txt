"Enter a float (e.g., -2.71): "					"Enter a float (e.g., -2.71): "

---- Completed parsing ----					---- Completed parsing ----

 Value of the expression is: 				      |	Value of the expression is:  -2.71
