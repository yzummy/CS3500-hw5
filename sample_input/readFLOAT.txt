# The expected output file expects you to enter -2.71
{
  print("Enter a float (e.g., -2.71): ");
  read()
}
