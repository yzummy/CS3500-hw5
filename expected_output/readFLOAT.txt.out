"Enter a float (e.g., -2.71): "

---- Completed parsing ----

Value of the expression is:  -2.71
